### PR TITLE
[-] Technical - Use handlebars for all templates

### PR DESCRIPTION
**Pull request by rap2hpoutre**
I had to refactor code a little to avoid some duplication: complexity has been reduced. This PR has to be tested manually, by creating some lumber projects.

### CODE REVIEW

**Comment by rap2hpoutre** [comment]
[Lumber - Use handlebars in all Lumber templates.](https://trello.com/c/8NMVT9Yh/4035-lumber-use-handlebars-in-all-lumber-templates)

File: templates/app/docker-compose.hbs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      context: .
+      dockerfile: Dockerfile
+    container_name: {{ containerName }}
+    environment:
+      - APPLICATION_PORT={{ port }}
+      - DATABASE_URL={{ databaseUrl }}
+{{#if dbSchema }}
+      - DATABASE_SCHEMA={{ dbSchema }}
+{{/if}}
+      - DATABASE_SSL={{ ssl }}
```
[the remark this answers is not in the document]
Comment (rap2hpoutre) [reply templates/app/docker-compose.hbs]: This has been set here: https://github.com/ForestAdmin/lumber/pull/381/files/86a3276b26f926509bd9959a346d8888dfeed301#diff-133bbb0f7ab282441fcb84d3faa6e9afR251.

I find it more readable like this (minimize interaction in template, prepare things in "controller").

File: templates/migrate/field-extend.txt [hunk 2]
```diff
@@ -1,4 +0,0 @@
-    '<%= field.name %>': {
-      type: DataTypes.<%= field.type %>,<% if (field.primaryKey) { %>
-      primaryKey: true <% } %>
-    },
```
[the remark this answers is not in the document]
Comment (rap2hpoutre) [reply templates/migrate/field-extend.txt]: It seems outdated:
 - https://github.com/ForestAdmin/lumber/blob/c68860210526edad5e49dd7d489e34da2790f449/templates/migrate/field-extend.txt
- https://github.com/ForestAdmin/lumber/search?q=field-extend&unscoped_q=field-extend

File: services/dumper.js [hunk 3]
```diff
@@ -33,6 +33,21 @@ function Dumper(config) {
     writeFile(to, fs.readFileSync(newFrom, 'utf-8'));
   }
 
+  function copyHandleBarsTemplate(args) {
```
[the remark this answers is not in the document]
Comment (rap2hpoutre) [reply services/dumper.js]: ❤️ ❤️ ❤️

File: templates/app/models/index.hbs [hunk 4]
```diff
@@ -17,28 +17,28 @@ let databaseOptions = {
   pool: { maxConnections: 10, minConnections: 1 },
   dialectOptions: {}
 };
+{{isMySQL}}
```
[the remark this answers is not in the document]
Comment (rap2hpoutre) [reply templates/app/models/index.hbs]: Good catch!!

File: services/dumper.js [hunk 5]
```diff
@@ -303,16 +290,21 @@ function Dumper(config) {
...
-
     writeAppJs();
     writeDockerCompose();
     writeDockerfile();
     writePackageJson();
+
+    // NOTICE: Copy simple templates files without replacements.
+    copyTemplate('server.hbs', `${path}/server.js`);
+    copyTemplate('views/index.hbs', `${path}/views/index.html`);
+    copyTemplate('middlewares/forestadmin.hbs', `${path}/middlewares/forestadmin.js`);
```
[the remark this answers is not in the document]
Comment (rap2hpoutre) [reply services/dumper.js]: Thanks!!

**Comment by rap2hpoutre** [comment]
@SteveBunlon Thanks to you, we avoided a few bugs!! Could you test again?

File: templates/app/docker-compose.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (SteveBunlon) [reply templates/app/docker-compose.hbs]: Agreed 👍

File: templates/migrate/field-extend.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (SteveBunlon) [reply templates/migrate/field-extend.txt]: Nice that you removed it 👍

**Pull request merged**